internal/runbits/runtime/progress/progress.go

```diff
@@ -344,12 +344,9 @@ Progress bars status:
 Still expecting:
  - Builds: %v
  - Downloads: %v
- - Installs: %v
-Event log:
-%s`,
```

Comment:
Log was being flooded, and the values weren't helpful.

Reply:
Would we want some sort of environment variable or other configuration option to get these logs?